src/_scss/components/_newBadge.scss

```diff
@@ -26,4 +26,43 @@
     margin-left: rem(10);
 }
 
+.covid-newbadge-dot-mobile {
+  height: rem(15);
+  width: rem(15);
+  margin-left: rem(8);
+  margin-bottom: rem(2);
+  background-color: $color-secondary-dark;
+  border-radius: 50%;
+  display: inline-block;
+}
+
+.outter {
```

Comment:
I would like to make these class names more specific and nested to prevent unintended style leakage (it's possible that something as generic as  `outer`, `inner`, or `white` could be used for another unrelated element)